### PR TITLE
fix: Minio service

### PR DESCRIPTION
**Pull request by Ajay-Maury**

### CODE REVIEW

File: src/minio/minio.service.ts [hunk 1]
```diff
@@ -33,7 +30,7 @@ export class MinioService {
     try {
       await this.minioClient.putObject(bucketName, objectName, await zipFile.toBuffer()); 
       const protocol = this.enableSSL ? 'https' : 'http';
-      return `${protocol}://${this.minioUrl}:${this.minioPort}/${bucketName}/${objectName}`
+      return `${protocol}://${this.minioUrl}/${bucketName}/${objectName}`
```
Comment (tushar5526) [inline src/minio/minio.service.ts:33]: Just use `${this.minioUrl}/${...}`. I don't we need `MINIO_USE_SSL`

Comment (tushar5526) [reply src/minio/minio.service.ts:33]: @charanpreet-s why do we need `MINIO_USE_SSL` ?

**Review by tushar5526 — APPROVED** [review]

File: src/minio/minio.service.ts (hunk 1 above)
Comment (charanpreet-s) [reply src/minio/minio.service.ts]: if in the case of staging, it is not required, minio has exposed this as an option. Initially, Minio was not secured on staging. @tushar5526

Comment (tushar5526) [reply src/minio/minio.service.ts]: Got it we have moved to SSL now. @Ajay-Maury you can merge it.

**Pull request merged**